src/screens/ToDoList/ToDoListItem/styl.scss

```diff
@@ -0,0 +1,63 @@
+@mixin row-form {
+  display: flex;
+  flex-direction: row;
+}
+
+@mixin button-check-delete {
+  border-radius: 50%;
+  outline: 0px;
+  cursor: pointer;
+  margin-left: 5px;
+  height: 17px;
+  width: 17px;
+}
```

Comment:
The naming of this mixin is not unfortunate - it points to very specific class when mixins should be generic.